codespell_lib/data/dictionary.txt

```diff
@@ -41539,6 +41550,15 @@ querried->queried
 querries->queries
 querry->query, quarry,
 querrying->querying
+querst->quest
```

Comment:
I assume `querist` is too rare, isn't it?

Reply:
`querist` is too rare